src/backend/distributed/utils/citus_outfuncs.c

```diff
@@ -173,9 +173,9 @@ OutMultiTreeRoot(OUTFUNC_ARGS)
 
 
 void
-OutMultiPlan(OUTFUNC_ARGS)
+OutDistributedPlan(OUTFUNC_ARGS)
 {
-	WRITE_LOCALS(MultiPlan);
+	WRITE_LOCALS(DistributedPlan);
 
 	WRITE_NODE_TYPE("MULTIPLAN");
```

Comment:
perhaps change this to DistributedPlan too ? It may trigger a lot more changes if we want to move from ```MultiTree``` or ```MultiNode``` notation to ```DistributedTree/Node``` notation.